src/contrib/cluster/Akka.Cluster.Sharding/ShardedDaemonProcess.cs

```diff
@@ -5,34 +5,41 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
+#nullable enable
```

Comment:
Every time I make API updates to an existing area of Akka.NET I always try to enable `nullable` so it'll be less jarring when we make it the default in the future.